BatteryLimits.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+public static class BatteryLimits
```

Comment:
initially this class holds two language specific const strings ......In the feature if we need to support more language features imagine the size of the class